Topics/User_Experience.md

```diff
@@ -24,6 +24,9 @@ ___Color Theory___
 ___Responsive Design___
 - [Responsive Design](https://devrix.com/tutorial/important-responsive-design/): Demonstrates important and benefits of responsive design, flexiblity they provide, easiblity for the consumers upon making the said software. 
 
+___ Nielsen's 10 Usability Heuristics___
```

Comment:
Fix the formatting here

Reply:
Fixed.

Reply:
I see there's a space already between Line 26 and Line 27. What needs to be fixed there? Not entirely sure.

Reply:
Removed the extra space now.

Reply:
Looks good, thanks